test/lib/test_runner.go

```diff
@@ -178,9 +184,81 @@ func Setup(t *testing.T, runInParallel bool, options ...SetupClientOption) *Clie
 	return client
 }
 
-func makeK8sNamespace(baseFuncName string) string {
-	base := helpers.MakeK8sNamePrefix(baseFuncName)
-	return names.SimpleNameGenerator.GenerateName(base + "-")
+func CreateNamespacedClient(t *testing.T) (*Client, error) {
```

Comment:
FYI: we intend to delete this lib and have it be replaced with `knative-sandbox/reconciler-test`

Reply:
Yeah. OK. But the previous attempt that I abandoned because of this new framework coming was 11 months ago (https://github.com/knative/eventing/pull/3313). If I understand it correctly, the majority of tests are still using this old style so it would be good to have some solution until we actually migrate everything to the new framework.